public/database.php

```diff
@@ -11,6 +13,13 @@ class Database {
 	 */
 	public function query($query) {
 		$mysqli = new mysqli("localhost", self::USERNAME, self::PASSWORD, "doomsday_forum");
-		return $mysqli->query($query);
+		$result = $mysqli->query($query);
+		$this->lastInsertedId = $mysqli->insert_id;
+
+		return $result;
+	}
+
+	public function getLastInsertedId() {
+		return $this->lastInsertedId;
```

Comment:
This class exists to abstract away the details of performing database operations.